sources/bloodhound/src/bin/bottlerocket-checks/checks.rs

```diff
@@ -180,3 +180,29 @@ impl results::Checker for BR01040300Checker {
         }
     }
 }
+
+// =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<=
+
+pub struct BR01040400Checker {}
+
+impl results::Checker for BR01040400Checker {
+    fn execute(&self) -> results::CheckerResult {
+        check_output_contains!(
+            SYSCTL_CMD,
+            ["user.max_user_namespaces"],
```

Comment:
hmm, is this a check in the CIS benchmark that must pass? It will be fun to revisit this once this feature moves to stable:

https://kubernetes.io/blog/2022/10/03/userns-alpha/

Reply:
Yeah, I think basically it means that folks requiring level 2 compliance will not be able to take advantage of user namespaces. Along with a few other things. Level 2 compliance is definitely more restrictive, so I would guess folks will not use it unless they are required.